instructors.js

```diff
@@ -0,0 +1,49 @@
+const Teachers = [{
```

Comment:
Great design @waseemqureshi2020 🤝

- [ ] Kindly make sure these speakers are displayed in two columns as provided in the design template. This is because the design should be a replica of the design provided.
- Your design
![your speakers](https://user-images.githubusercontent.com/63917725/197341471-5c7352f4-092f-4a02-91f8-abc72ddce94e.PNG)
- Provided design
![provided-design](https://user-images.githubusercontent.com/63917725/197341498-919ceb34-5d6d-45d9-a94a-c5430d3144e4.PNG)

Reply:
I know but you are comparing tablet view with desktop, According to the requirements, there are only two versions, one for Mobile and one for desktop. The resolution for the mobile version should be 768px or less and for the desktop version, it should be more than 768px.